raft/src/main/java/org/apache/kafka/snapshot/SnapshotWriter.java

```diff
@@ -55,7 +60,7 @@
      * @param compressionType the compression algorithm to use
      * @param serde the record serialization and deserialization implementation
      */
```

Comment:
Let's remove this Java Doc since you moved it to the static method and the constructor is private.

Reply:
ack